site/src/app/v2/trouver-un-club/components/club-list-view/ClubListView.tsx

```diff
@@ -0,0 +1,82 @@
+import { SportGouvJSONRecordsResponse } from 'types/Club';
+import ClubCount from '../club-count/ClubCount';
+import styles from './styles.module.scss';
+import cn from 'classnames';
+import Card from '@codegouvfr/react-dsfr/Card';
+import Badge from '@codegouvfr/react-dsfr/Badge';
+import Tag from '@codegouvfr/react-dsfr/Tag';
+import { DisabilityTag } from '../disability-tag/DisabilityTag';
+import Button from '@codegouvfr/react-dsfr/Button';
+import { usePathname } from 'next/navigation';
+
+interface Props {
+  clubs: SportGouvJSONRecordsResponse;
+  onSeeMoreClubsClicked: () => void;
+}
+const ClubListView = ({ clubs, onSeeMoreClubsClicked }: Props) => {
+  const pathname = usePathname();
+  const isLastPage = clubs.total_count === clubs.results.length;
+
+  return (
+    <div>
```

Comment:
Pas besoin d'un div, un fragment suffit
```suggestion
    <>
```